.gitignore

```diff
@@ -18,11 +18,17 @@
 *.sublime-workspace
 
 # IDE - VSCode
-.vscode/*
+*/vscode/*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+**/allure-results/**
+**/tests/**
+as-a-user.json
```

Comment:
What's as-a-user.json? Should we be ignoring it?

Reply:
this json seems is pulling data for the cucumber html results

Reply:
This is under the allure results? You may need to remove it from git ignore, delete the allure folder, commit that, then add it back into git ignore. I'll do it in my branch if it's problematic

Reply:
Got it now, I will do it, thanks @mdp0